### ISSUE TITLE
"Search all species" checkbox on redet window is obscuring it's own label

### ISSUE THREAD

**Issue by Sam-Amy** [BiologicalRecordsCentre/iRecord#1721]
"Search all species" checkbox on redet window for species taken from a limited list is obscuring it's own label:
![image](https://github.com/user-attachments/assets/23ef6bd5-fdbb-4576-8e1a-1a8f9e9eac19)

@kitenetter do you know in which cases this is an option? I would have expected for example all NPMS wildflower and indicator surveys to give this option, given they are taken from a limited list but this doesn't always seem to be the case though. In fact, I can't even find an example now!

**Comment by kitenetter** [comment]
@Sam-Amy my understanding is that the "search all species" option only appears if the record in question is from a survey that uses a custom (non-UKSI) species dictionary. Main example I'm aware of is NatureSpot, also National Moth Night and a few others.

Does NPMS use a custom dictionary, or just a cut-down version of UKSI? If the latter then I think that would explain why the tick box doesn't appear (it would only be needed if an NPMS record needed to be redetermined to something other than a plant, which I guess is unlikely).

Having the tick box overlapping is not ideal but doesn't actually cause anything to fail.

**Comment by sacrevert** [comment]
It uses a cut-down of the UKSI, but I agree with Sam that the "seach all species" option does sometimes appear for the redetermination of NPMS records (which is useful, as those records made at the Wildflower and Indicator levels cannot be redetermined to species outside of those lists otherwise, at least that certainly was the case). These experiences don't seem to agree with your understanding @kitenetter ?

**Comment by kitenetter** [comment]
Which probably means my understanding is wrong! But I don't see why the option sometimes appears for NPMS but not always.

**Comment by sacrevert** [comment]
@Sam-Amy are you sure the cases where it doesn't appear are not just Inventory level (or Extra species survey) records? I just checked a couple and it appears as expected for Indicator survey (32|155) but not for Inventory (32|154), again, as expected

**Comment by Sam-Amy** [comment]
I have been trying indicator (32|155), and wildflower (32|87) without seeing this e.g. 38008444, 38007529, 37986145, 37898554.

I'm expecting it to be visible as soon as I click on the redetermination icon and the window comes up, is that right, or have I forgotten something?

**Comment by sacrevert** [comment]
Yes, that is baffling. I looked at 38008444, and I would expect the checkbox to be visible, as it is an Indicator survey species with no prior redet. Looks as though redet options are entirely unrestricted. No idea what's determining whether the Search all species checkbox appears in that case!

**Comment by sacrevert** [comment]
The issue with the checkbox obscuring its label is still current. From today:

<img width="2299" height="1270" alt="Image" src="https://github.com/user-attachments/assets/5c9d58f1-771b-4bf2-802f-f66c9ecaa394" />
